src/definition/exceptions/Toast.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * The custom exceptions from apps
+ *
+ * It's used to display toasts to users that * a _known_
+ * exception has happened during the execution of the apps.
+ */
+class ErrorToast extends Error { }
+export const Toast = {
```

Comment:
Other exceptions for similar purposes follow a different naming pattern, as `FileUploadNotAllowedException` for instance (https://github.com/RocketChat/Rocket.Chat.Apps-engine/blob/refactor-bridges/src/definition/exceptions/FileUploadNotAllowedException.ts).

Reply:
I still think the name is not appropriate for the error. Naming it `Toast` with an internal `ErrorToast` prop ties it too much to how the host system will implement the error display _and_ can be misleading in the sense that it might make you think other errors _won't be shown as a toast_, which is not the case.

I'd suggest following the `FileUploadNotAllowedException` pattern, and name the exported class as `InvalidSettingValueException` for instance